docs/index.bs

```diff
@@ -1809,42 +1809,48 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
     <section algorithm="cache-put">
       <h4 id="cache-put">{{Cache/put(request, response)}}</h4>
 
-      <dfn method for="Cache"><code>put(|request|, |response|)</code></dfn> method *must* run these steps:
-
-        1. Let |r| be null.
-        1. If |request| is a {{Request}} object, then:
-            1. Set |r| to |request|'s [=Request/request=].
-            1. If |r|'s [=request/url=]'s [=url/scheme=] is not one of "`http`" and "`https`", or |r|'s [=request/method=] is not \`<code>GET</code>\`, return [=a promise rejected with=] a `TypeError`.
-        1. Else if |request| is a string, then:
-            1. Set |r| to the associated [=Request/request=] of the result of invoking the initial value of {{Request}} as constructor with |request| as its argument. If this [=throws=] an exception, return [=a promise rejected with=] that exception.
-            1. If |r|'s [=request/url=]'s [=url/scheme=] is not one of "`http`" and "`https`", return [=a promise rejected with=] a `TypeError`.
-        1. If |response|'s associated [=Response/response=]'s [=response/status=] is `206`, return [=a promise rejected with=] a `TypeError`.
-        1. If |response|'s associated [=Response/response=]'s [=response/header list=] contains a <a>header</a> [=header/named=] \`<code>Vary</code>\`, then:
+      <dfn method for="Cache"><code>put(|requestArg|, |responseObject|)</code></dfn> method *must* run these steps:
```

Comment:
Using |requestArg| and |requestObject| so |request| and |response| can be the concepts rather than the object.

This cleared up a lot of misuse of types in this algorithm.

Reply:
requestObject for consistency? (I've also never seen arg used so explicitly.)

Reply:
It might be a string (see 1.2).